deploy/eck-stack/examples/agent/fleet-agents.yaml

```diff
@@ -28,26 +24,18 @@ eck-kibana:
   #
   fullnameOverride: kibana
   
-  # Version of Kibana.
-  #
-  version: 8.5.0
-  
   spec:
-    # Count of Kibana instances to create.
-    #
-    count: 1
-  
     # Reference to ECK-managed Elasticsearch instance, ideally from {{ "elasticsearch.fullname" }}
     #
     elasticsearchRef:
       name: elasticsearch
 
     config:
       # Note that these are specific to the namespace into which this example is installed, and are
-      # using `default` as configured here.  If installed outside of the `default` namespace,
+      # using `elastic-stack` as configured here.  If installed outside of the `elastic-stack` namespace,
```

Comment:
I was looking where is configured the `elastic-stack` namespace and IIUC it's from README.md:L40 when we explain how to use the chart `helm install es-kb-quickstart elastic/eck-stack -n elastic-stack --create-namespace`. Ok, but it's a bit tricky if you look only at the yaml file.